docs/docs-content/clusters/public-cloud/aws/eks.md

```diff
@@ -39,118 +44,144 @@ The following tags should be added to the virtual private network (VPC) public s
+8. Specify your preferred **OS Patching Schedule** for EKS-managed machines.
 
-:::
+9. Enable any scan options you want Palette to perform, and select a scan schedule. Palette provides support for Kubernetes configuration security, penetration testing, and conformance testing.
 
-12. An optional taint label can be applied to a node pool during the cluster creation. For a an existing cluster, the taint label can be edited, review the [Node Pool](../../cluster-management/node-pool.md) management page to learn more. Toggle the **Taint** button to create a label.
+10. Schedule any backups you want Palette to perform. Review [Backup and Restore](../../cluster-management/backup-restore/backup-restore.md) for more information.
 
+11. RBAC configuration is required when you configure OIDC. You must map a set of users or groups to a Kubernetes RBAC role. To learn how to map a Kubernetes role to users and groups, refer to [Create Role Bindings](../../cluster-management/cluster-rbac.md/#create-role-bindings). Refer to [Use RBAC with OIDC](../../../integrations/kubernetes.md/#use-rbac-with-oidc) for an example.
 
-13. Enable or disable node pool taints. If tainting is enabled then you need provide values for the following parameters:
-    
-    |**Parameter**| **Description**|
-    |-------------|---------------|
-    |**Key**      |Custom key for the taint.|
-    |**Value**    | Custom value for the taint key.|
-    | **Effect**  | Make the choice of effect from the drop-down menu. Review the effect table bellow for more details. |
+12. Click on the **Validate** button and review the cluster configuration and settings summary. 
+
+13. Click **Finish Configuration** to deploy the cluster. 
+
+  The cluster details page of the cluster contains the status and details of the deployment. Use this page to track the deployment progress.
   
-    #### Effect Table
-    
-    |**Parameter**| **Description**|
-    |-------------|---------------|
-    | **NoSchedule**|  A pod that cannot tolerate the node taint and should not be scheduled to the node. 
-    | **PreferNoSchedule**| The system will avoid placing a non-tolerant pod to the tainted node but is not guaranteed.
-    | **NoExecute**|  New pods will not be scheduled on the node, and existing pods on the node if any on the node will be evicted they do not tolerate the taint. |
+  :::info
+  
+  Provisioning an AWS EKS clusters can take several minutes.
+  
+  :::
 
-14. Click on **Next**.  
-    
-15. The settings page is where you can configure patching schedule, security scans, backup settings, setup role based access control (RBAC), and enable [Palette Virtual Clusters](../../../devx/palette-virtual-clusters/palette-virtual-clusters.md). Review the settings and make changes if needed. Click on **Validate**.
 
-16. Review the settings summary and click on **Finish Configuration** to deploy the cluster. Be aware that provisioning an AWS EKS clusters can take several minutes.

```

Comment:
A question we see a lot is how to connect to the cluster. We could add a sentence or two about this and link to the kubectl page we have is cluster management.

Reply:
Added sentence with a link.